CHANGELOG.md

```diff
@@ -1429,7 +1429,7 @@ Features for this release were tracked via the use of the [kubernetes/features](
 
 	* Automate configuring nameserver in cluster-dns for CoreDNS provider ([#42895](https://github.com/kubernetes/kubernetes/pull/42895), [@shashidharatd](https://github.com/shashidharatd))
 
-	* A new controller for managing DNS records is introduced which can be optionally disabled to enable third party components to manage DNS records for federated services. ([#450354](https://github.com/kubernetes/kubernetes/pull/450354), [@shashidharatd](https://github.com/shashidharatd)(https://github.com/shashidharatd)])
```

Comment:
Good catch! Seems like this line mis-matches with my regex in #48328, so I missed it in that PR.
/cc @luxas